File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int big = 0, cnt = 0, n;
        for(int i = 1; i <= 9; i++){
            n = in.nextInt();
            if(n > big){
                big = n;
                cnt = i;
            }
        }
        System.out.printf("%d\n%d", big, cnt);
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var big = 1; var index = 0
    for(i in 1..9){
        var n = sc.nextInt()
        if(big < n){
            big = n
            index = i
        }
    }
    print("$big\n$index")
}